pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -1002,6 +1017,7 @@ https://github.com/tikhomirov/vim-glsl/,,
 https://github.com/jamessan/vim-gnupg/,,
 https://github.com/fatih/vim-go/,,
 https://github.com/rhysd/vim-grammarous/,,
+https://github.com/bytesnake/vim-graphical-preview/,HEAD,
```

Comment:
since this uses rust, it likely won't work without an override. you can take a look at `sg-nvim` or `vim-clap` for inspiration

to test it, you need to run the update script again due to #232078

Reply:
Think I added the overrides correctly and I ran `update.py` again